services/agora/src/pages/moderate/comment/[commentSlugId]/index.vue

```diff
@@ -1,46 +1,63 @@
 <template>
-  <div class="container">
-    <div class="title">Moderate the opinion "{add opinion title excerpt}"</div>
-
-    <q-select
-      v-model="moderationAction"
-      :options="actionMapping"
-      label="Action"
-      emit-value
-      map-options
-    />
-
-    <q-select
-      v-model="moderationReason"
-      :options="reasonMapping"
-      label="Reason"
-      emit-value
-      map-options
-    />
-
-    <q-input v-model="moderationExplanation" label="Explanation (optional)" />
-
-    <ZKButton
-      v-if="!hasExistingDecision"
-      label="Modify"
-      color="primary"
-      @click="clickedSubmit()"
-    />
-    <ZKButton
-      v-if="hasExistingDecision"
-      label="Moderate"
-      color="primary"
-      @click="clickedSubmit()"
-    />
-
-    <ZKButton
-      v-if="hasExistingDecision"
-      label="Withdraw"
-      color="secondary"
-      text-color="primary"
-      @click="clickedWithdraw()"
-    />
-  </div>
+  <MainLayout
+    :general-props="{
+      addBottomPadding: false,
+      enableHeader: true,
+      enableFooter: true,
+      reducedWidth: false,
+    }"
+    :menu-bar-props="{
+      hasBackButton: false,
+      hasSettingsButton: false,
+      hasCloseButton: true,
+      hasLoginButton: true,
+    }"
+  >
+    <div class="container">
+      <div class="title">
+        Moderate the opinion "{add opinion title excerpt}"
```

Comment:
you're going to fix that as part of another PR? see full request for changes:
https://github.com/zkorum/agora/pull/135#issuecomment-2573544642